demos/python_demos/llm_text_generation/README.md

```diff
@@ -50,46 +106,78 @@ Depending on the use case, `./servable_unary` and `./servable_stream` showcase d
+It is difficult to say how many helicopters human can eat in one sitting without knowing what type of person you are referring to. You may want to ask someone who knows about this topic for an accurate response to this question. However, typically speaking, it would be impossible for a human to consume an entire aerial vehicle, consisting of multiple compartments and rotors, every day if they lived to be 100 years old. However, humans can ingest larger quantities of food, like energy bars or canned goods, which have a smaller volume and can be consumed over a period of time, making it easier for them to consume large amounts of food at once. It is also possible that some people are able to consume helicopter parts due to their exceptional strength, stamina, endurance, or aversion to dehydration, among other reasons.
+
+Total time 11662 ms
 ```
 
-## Requesting the LLM with gRPC streaming
+## Use LLM model with gRPC streaming
 
+### Deploy OpenVINO Model Server with the Python Calculator
 
-Start the Model Server with different directory mounted (`./servable_stream`).
+The model server can be deployed with our streaming example by just mounting different workspace location from `./servable_stream`.
 It contains modified `model.py` script which yields the intermediate results instead of returning it at the end of `execute` method.
-The `graph.pbtxt` is also modified to include cycle in order to make the Python Calculator run in a loop.  
+The `graph.pbtxt` is also modified to include a cycle in order to make the Python Calculator run in a loop.  
 
 ```bash
-docker run -it --rm -p 9000:9000 -v ${PWD}/servable_stream:/workspace -v ${PWD}/model:/model openvino/model_server:py --config_path /workspace/config.json --port 9000
+docker run -d --rm -p 9000:9000 -v ${PWD}/servable_stream:/workspace -v ${PWD}/${SELECTED_MODEL}:/model \
+-e SELECTED_MODEL=${SELECTED_MODEL} openvino/model_server:py --config_path /workspace/config.json --port 9000
 ```
 
+Like with the unary example, you can also deploy the compressed model by just changing the model path mounted to the container. For example:
+```bash
+docker run -d --rm -p 9000:9000 -v ${PWD}/servable_stream:/workspace -v ${PWD}/${SELECTED_MODEL}_INT8_compressed_weights:/model \
+-e SELECTED_MODEL=${SELECTED_MODEL} openvino/model_server:py --config_path /workspace/config.json --port 9000
+```
+> **Note** Check the docker container logs to confirm the model is loaded before running the client. Depending on the model and hardware it might take up to several minutes.
+
+## Running the client with the LLM model and gRPC streaming
+
 Run time streaming client `client_stream.py`:
 ```bash
-python3 client_stream.py --url localhost:9000 --prompt "Describe the state of the healthcare industry in the United States"
+python3 client_stream.py --url localhost:9000 --prompt "How many helicopters can a human eat in one sitting?"
```

Comment:
@dtrawins can we change this prompt to something more realistic? I suggest replacing helicopters with apples or some food that people would eat. Or create a new prompt.